pinot-common/src/test/resources/schemaTest.schema

```diff
@@ -88,6 +88,12 @@
       "format": "1:DAYS:SIMPLE_DATE_FORMAT:yyyyMMdd",
       "granularity": "1:DAYS",
       "defaultNullValue": "20200101"
+    },
+    {
+      "name": "dateTime3",
+      "dataType": "TIMESTAMP",
+      "format": "1:MILLISECONDS:TIMESTAMP",
```

Comment:
Just checked a community discussion that this can only be TIMESTAMP and setting the format to 
`1:MILLISECONDS:EPOCH` will fail. 

Is there a reason why? I think we should support both.

Reply:
I grabbed it from here: https://github.com/apache/pinot/blob/master/pinot-tools/src/main/resources/examples/batch/githubEvents/githubEvents_schema.json#L36